plugins/catalog-backend-module-github-org/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@backstage/plugin-catalog-backend-module-github-org",
+  "description": "The github-org backend module for the catalog plugin.",
+  "version": "0.0.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.cjs.js",
+    "types": "dist/index.d.ts"
+  },
+  "backstage": {
+    "role": "backend-plugin-module"
+  },
+  "scripts": {
+    "start": "backstage-cli package start",
+    "build": "backstage-cli package build",
+    "lint": "backstage-cli package lint",
+    "test": "backstage-cli package test",
+    "clean": "backstage-cli package clean",
+    "prepack": "backstage-cli package prepack",
+    "postpack": "backstage-cli package postpack"
+  },
+  "dependencies": {
```

Comment:
Best trim these deps a bit

Reply:
Done in https://github.com/backstage/backstage/pull/20361/commits/2b3c56a5d58c1df493444d9a69ac9015811e3384

Found with knip:

```sh
npx knip --workspace plugins/catalog-backend-module-github-org
....
Unused dependencies (8)
@backstage/catalog-model       plugins/catalog-backend-module-github-org/package.json
@backstage/integration         plugins/catalog-backend-module-github-org/package.json
@backstage/plugin-events-node  plugins/catalog-backend-module-github-org/package.json
@octokit/graphql               plugins/catalog-backend-module-github-org/package.json
lodash                         plugins/catalog-backend-module-github-org/package.json
uuid                           plugins/catalog-backend-module-github-org/package.json
winston                        plugins/catalog-backend-module-github-org/package.json
```